### PR TITLE
Update tutorial documentation

### PR DESCRIPTION
**Pull request by gwaters**
When following the tutorial documentation (https://docs.saltstack.com/en/latest/topics/tutorials/minionfs.html) I found that I couldnt run the cp.push command. So I wanted to update the docs to show what needed to be enabled first then running the command. I am very open to changing whatever doesnt look good. @gravyboat

### CODE REVIEW

File: doc/topics/tutorials/minionfs.rst [hunk 1]
```diff
@@ -136,4 +156,4 @@ Or we can use a more elegant and salty way to add an SSH key:
 
 
 
-.. [*] Yes, that was the actual key on my server, but the server is already destroyed.
\ No newline at end of file
+.. [*] Yes, that was the actual key on my server, but the server is already destroyed.
```
Comment (gravyboat) [inline doc/topics/tutorials/minionfs.rst]: Is this line supposed to exist?

Comment (gwaters) [inline doc/topics/tutorials/minionfs.rst]: Thats not from me, I think the original author of that comment didnt want to get any lip about showing his key / keyart for the demonstration.

File: doc/topics/tutorials/minionfs.rst [hunk 2]
```diff
@@ -4,10 +4,27 @@
 MinionFS Backend Walkthrough
 ============================
 
+Propagating Files
+============================
```
Comment (gravyboat) [inline doc/topics/tutorials/minionfs.rst]: We can cut down on the number of `=` signs, they just need to be the same length as the line they are below so the character count is the same.

File: doc/topics/tutorials/minionfs.rst [hunk 3]
```diff
@@ -4,10 +4,27 @@
...
+
 .. versionadded:: 2014.1.0
 
 Sometimes, you might need to propagate files that are generated on a minion.
-Salt already has a feature to send files from a minion to the master:
+Salt already has a feature to send files from a minion to the master.
+
+Enabling Feature
```
Comment (gravyboat) [inline doc/topics/tutorials/minionfs.rst]: What do you think about changing this to `Enabling File Propagation`?

File: doc/topics/tutorials/minionfs.rst [hunk 4]
```diff
@@ -4,10 +4,27 @@
...
+====================
+
+To use the propagate feature one configuration change is required on the
```
Comment (gravyboat) [inline doc/topics/tutorials/minionfs.rst]: I think this and line 19 could be worded slightly better, noting that `one configuration change is required` isn't necessary. Instead just say what the change and that it lives in the master config.

File: doc/topics/tutorials/minionfs.rst (hunk 1 above)
Comment (gravyboat) [inline doc/topics/tutorials/minionfs.rst]: Ahh okay.

**Comment by gravyboat** [comment]
I added a couple line comments, let me know what you think, other than those few things it looks good to me!

**Comment by gwaters** [comment]
Added your suggestions, they are good ones. Thanks!

File: doc/topics/tutorials/minionfs.rst [hunk 5]
```diff
@@ -4,10 +4,27 @@
...
+Propagating Files
+=================
+
 .. versionadded:: 2014.1.0
 
 Sometimes, you might need to propagate files that are generated on a minion.
-Salt already has a feature to send files from a minion to the master:
+Salt already has a feature to send files from a minion to the master.
+
+Enabling File Propation
```
Comment (bechtoldt) [inline doc/topics/tutorials/minionfs.rst]: Propation? :)

**Comment by cachedout** [comment]
Thanks, @gwaters !

File: doc/topics/tutorials/minionfs.rst (hunk 5 above)
Comment (gwaters) [inline doc/topics/tutorials/minionfs.rst]: Doh!
Do I need to do another PR or did someone see your comment as well and fix it ?
